frontend-plugin-core/src/main/java/com/github/eirslett/maven/plugins/frontend/lib/YarnRunner.java

```diff
@@ -59,6 +60,16 @@ private static List<String> buildArguments(final YarnExecutorConfig config, Prox
     }
 
     /**
+     * <p>
+     * Running {@code yarn node --version} in yarn classic gives us an output like this:
+     * <pre>
+     * yarn node v1.22.22
+     * v20.10.0
+     * Done in 0.02s.
+     * </pre>
+     * while yarn berry will give us just the output
+     * </p>
+     * <p>
      * Running {@code yarn versions} gives an output like this:
```

Comment:
This wording became a bit redundant/awkward?

Reply:
If you have a suggestion, I'm happy to take it? Both outputs are listed because we have two commands we could be running.

Reply:
maybe just try running --version on node and yarn and avoid all the variance?

Reply:
Caught up in person to clear the confusion and added more to the comment